Jenkinsfile

```diff
@@ -0,0 +1,26 @@
+pipeline {
```

Comment:
Should this file goes to the `secure-backend` repo?

Reply:
This is for the jenkins job that monitors changes to this repo. You're right that users of this code, like padvisor in the secure-backend repo, might also want tests, but I think those tests would be higher-level and wouldn't just be running the unit tests from this repo.